cmd/src/testdata/search_formatting/basic-commit-new.test.json

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
You have accidently comitted a `cmd/src/src` binary file to this repo, remove that